From 8cb4b8e9b6a16364f67b07b77d1a487026ab54b5 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Wed, 28 Oct 2020 21:17:33 +0000
Subject: [PATCH] Re-enable CodeStyle for source-build

Add back building of CodeStyle projects for source-build.  Upstream fix
is at https://github.com/dotnet/roslyn/pull/48999.

---
 src/CodeStyle/Directory.Build.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CodeStyle/Directory.Build.props b/src/CodeStyle/Directory.Build.props
index 0991c17..3722c4d 100644
--- a/src/CodeStyle/Directory.Build.props
+++ b/src/CodeStyle/Directory.Build.props
@@ -1,7 +1,7 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
   <PropertyGroup>
-    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
 
     <!-- https://github.com/dotnet/roslyn/issues/46984 -->
     <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
-- 
1.8.3.1

